releases.go

```diff
@@ -306,6 +306,9 @@ func newServiceProcess(release *Release, p *Process) *scheduler.Process {
 	if len(ports) > 0 {
 		env["PORT"] = fmt.Sprintf("%d", *ports[0].Container)
 
+		// gliderlabs registrator
+		labels[fmt.Sprintf("SERVICE_%d_NAME", *ports[0].Container)] = release.App.Name
```

Comment:
Registrator [reads both env and labels](https://github.com/gliderlabs/registrator/blob/master/bridge/util.go#L33), and the former takes precedence over the latter. My goal is to implement something that can be overridden by config:set. You could of course also achieve this by taking care not to clobber SERVICE_NAME passed by config. While the env implementation adds some pollution to the app (which was another motivator), it might on second thought also be a bit confusing using a label because there will (if overridden) be two values present upon container inspection.